react/src/components/AssetDetail/AssetDetail.tsx

```diff
@@ -0,0 +1,172 @@
+        {fileType ? (
+          <>
+            <Suspense fallback={<LoadingSpinner />}>
+              <AssetRenderer type={fileType} source={featureSource} />
+            </Suspense>
+            <Button /*TODO Download Action */>Download</Button>
+          </>
+        ) : (
+          <>
+            <SectionMessage type="info">Feature has no asset.</SectionMessage>
+
+            <Button type="primary" /* TODO Add asset to a feature */>
+              Add asset from DesignSafe
+            </Button>
+          </>
+        )}
+      </div>
+      <div className={styles.bottomSection}>
+        <div className={styles.metadataTable}>
+          <table>
+            <thead>
+              <tr>
+                <th colSpan={2}>Metadata</th>
+              </tr>
+            </thead>
+            <tbody>
+              {selectedFeature?.properties &&
+              Object.keys(selectedFeature.properties).length > 0 ? (
+                Object.entries(selectedFeature.properties)
+                  .filter(([key]) => !key.startsWith('_hazmapper'))
+                  .sort(([keyA], [keyB]) => keyA.localeCompare(keyB)) // Alphabetizes metadata
+                  .map(([propKey, propValue]) => (
+                    <tr key={propKey}>
+                      <td>{_.startCase(propKey)}</td>
+                      <td>
+                        {propKey.startsWith('description') ? (
+                          <code>{propValue}</code>
+                        ) : (
+                          _.trim(JSON.stringify(propValue), '"')
+                        )}
+                      </td>
+                    </tr>
+                  ))
+              ) : (
+                <tr>
+                  <td colSpan={2}>
+                    There are no metadata properties for this feature.
+                  </td>
+                </tr>
+              )}
+            </tbody>
+          </table>
+          <table>
+            <thead>
+              <tr>
+                <th colSpan={2}>Geometry</th>
+              </tr>
+            </thead>
+            <tbody>
+              {selectedFeature?.geometry &&
+                Object.entries(selectedFeature.geometry)
+                .map(
+                  ([propKey, propValue]) => (
+                    propValue && propValue !== undefined && propValue.toString().trim() !== '' && propValue.toString() !== 'null' && (
+                    <tr key={propKey}>
+                      <td>{_.trim(_.startCase(propKey.toString()), '"')}</td>
+                      <td>
+                        {' '}
+                        {Array.isArray(propValue) && propValue.length === 2
+                          ? `Latitude: ${propValue[0].toString()},
+                             Longitude: ${propValue[1].toString()}`
+                          : _.trim(JSON.stringify(propValue), '"')}
+                      </td>
+                    </tr>
+                    )
+                  )
+                )}
```

Comment:
i think we gotta match https://github.com/TACC-Cloud/hazmapper/tree/main/angular/src/app/components/feature-geometry.  Works as is for single but then for geometries we gotta show some sort of geometry (area, bounding box etc).